modules/aleph/aleph.module

```diff
@@ -32,6 +32,19 @@ function aleph_get_branches() {
   );
 }
 
+/**
+ * Checks if a branch exists in aleph_get_branches().
+ *
+ * @param $branch
+ *   The branch code to check for.
+ *
+ * @return bool
+ *   True if the branch exists.
+ */
+function aleph_filter_items($branch) {
```

Comment:
Is there a reason why this is a actual function and not just an anonymous one defined when used in `AlephMaterialHandler`?

Reply:
See also comment below.